content/articles/getting-started-with-unit-testing-with-spring-boot/index.md

```diff
@@ -31,31 +28,31 @@ It's good to simulate database-interacting beans and disable Spring Boot test DB
 #### Services
 This is the layer where all the business logic is implemented. It is tested to make sure that the business logic is correct.
 ### How to test controllers
-Let us now look at how to test controllers in Spring Boot. To accomplish this, you will need to import some dependencies using the `Spring Initializer` and `IntelliJ IDEA`. `IntelliJ IDEA` lets us create a Spring Boot application using the Spring Initializr service. Open `file>new>project` and select `Spring Initializr` as shown below:
+Let us now look at how to test controllers in Spring Boot. To accomplish this, you will need to import some dependencies using the `Spring Initializer` and `IntelliJ IDEA`. In, `IntelliJ IDEA` lets us create a Spring Boot application using the Spring Initializr service. Open `file>new>project` and select `Spring Initializr` as shown below:
 
-![Spring Initializer](/engineering-education/getting-started-with-unit-testing-with-spring-boot/spring-Initializer.png)
+![Spring Initializer](/engineering-education/getting-started-with-unit-testing-with-spring-boot/spring-initializer.png)
 
-> Note: You can rename the Package name.
+> Note: I have renamed my Package name to `UnitTesting`.
 
-Click next to proceed. Type *web* to search the required dependencies in the search bar. Select *Spring Web*, *Spring Reactive Web*, *Spring Web Services*, and click *finish* to download the initializr template:
+Click next to proceed. Type *web* to search the required dependencies in the search bar. Select *Spring Web* and click *finish* to download the initializr template:
 
-![Spring Initializer](/engineering-education/getting-started-with-unit-testing-with-spring-boot/spring-Initializer-Dependency.png)
+![Spring Initializer](/engineering-education/getting-started-with-unit-testing-with-spring-boot/spring-initializer-dependency.png)
 
-We've prepared the environment successfully. Now we need to test the controller. In `/src/main/java` create a new package named `Controller`. Proceed to create a Java class named `HelloContoller.java` in  `/src/main/java/Controller`. Within that file, add the code below:
+We've prepared the environment successfully.🔥 Now we need to test the controller. In `/src/main/java/UnitTesting` create a new package named `controller`. Proceed to create a Java class named `HelloContoller.java` in  `/src/main/java/UnitTesting/controller`. Within that file, add the code below:
 ```java
-package controller;
+package UnitTesting.controller;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 @RestController
 public class HelloController {
     @GetMapping("/hello")
     public String hello(@RequestParam(name = "name", defaultValue = "Hello world") String name){
-        return String.format("Hello, %s", name );
+        return String.format(name);
     }
 }
 ```
```

Comment:
As I mentioned previously, packages containing your tests should be in `/src/test/java`